spk/gerrit/src/dsm-control.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+
+# Package
+PACKAGE="gerrit"
+DNAME="Gerrit"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PATH="${INSTALL_DIR}/bin:${GIT_DIR}/bin:${PATH}"
+USER="gerrit"
+PID_FILE="${INSTALL_DIR}/logs/gerrit.pid"
+LOG_FILE="${INSTALL_DIR}/logs/error_log"
+
+start_daemon ()
+{
+    su ${USER} -c "PATH=${PATH} ${INSTALL_DIR}/bin/gerrit.sh start"
```

Comment:
This file comes from the gerrit package itself and is created/overwritten by the init call. It's pretty much just a standard initscript in terms of format, but it sets up a bunch of config variables and stuff before launching the executable .war.

After init we have to sed a line in there during postinst, since it makes use of a `ps` arg format unsupported by DSM. Fortunately the start-stop-status commands all seem to work fine after this though, so just passing through to the Gerrit script saves a whole bunch of work.